ex5/ex5.c

```diff
@@ -16,7 +16,30 @@ char* msg3 = "hello world #3";
 
 int main(void)
 {
+    char inbuf[MSGSIZE];
+    int pipearr[2];  // Used to store two ends of first pipe 
+    if (pipe(pipearr) < 0) {
+        fprintf(stderr, "pipe failed\n");
+        exit(1);
+    }
     // Your code here
-    
+    int child = fork();
+
+    if (child == 0){
+        write(pipearr[1], msg1, MSGSIZE); 
+        write(pipearr[1], msg2, MSGSIZE); 
+        write(pipearr[1], msg3, MSGSIZE); 
+        close(pipearr[1]);
+    }
+    else{
+        for (int i = 0; i < 3; i++) {
+        // read 16 bytes of data from the read file descriptor 
+        read(pipearr[0], inbuf, MSGSIZE);
+        printf("% s\n", inbuf);
+        }
```

Comment:
I am getting a warning about line 38 - is that space between the % and s supposed to be there? It still works either way.

Reply:
Removed space, I have to wonder why I end up with less flags then you do